CONTRIBUTING.md

```diff
@@ -42,7 +48,26 @@ The following rule governs documentation contributions:
 
 * Contributions must be licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
 
-At least if you add a new documentation to the repository, add your name into the [PARTICIPANTS](./PARTICIPANTS.MD) file.
+* At least if you add a new documentation to the repository, add your name into the [PARTICIPANTS](./PARTICIPANTS.MD) file or send an onboarding email to _project email address_.
+
+## Creating Branches
+
+Branches allow you to develop features, fix bugs, or safely experiment with new ideas in a contained area of our repository. If you need a friendly video introduction to git branching, please try [part two of "GitHub for Poets"](https://www.youtube.com/watch?v=oPpnCh7InLY) on Youtube, you may also read GitHub's ["Creating and deleting branches within your repository"](https://docs.github.com/en/articles/creating-and-deleting-branches-within-your-repository) for a deep dive.
```

Comment:
Contributor should normally a fork the repository before creating a branch (fork and pull model). Only codeowners have the right to create branches within the repository (?).

Reply:
For the moment I would like to keep it as simple as it is and don't mention the forking. If we get questions for clarification we can add it later.

Reply:
Most contributors have only view rights on the repositories, so they must use fork/pull model to contribute. And: already the third video of "Git & Github for poet" explains how this works.

Reply:
Working with forks added